Custom Device Source/Scripting/Modules/Channel Classes/Analog Channel/Analog Channel.lvclass

```diff
@@ -0,0 +1,64 @@
+		<Property Name="NI.LibItem.Scope" Type="Int">2</Property>
+	</Item>
+	<Item Name="Properties" Type="Folder">
+		<Item Name="default value" Type="Property Definition">
+			<Property Name="NI.ClassItem.Property.LongName" Type="Str">default value</Property>
+			<Property Name="NI.ClassItem.Property.ShortName" Type="Str">default value</Property>
+			<Property Name="NI.ClassItem.Property.SortKey" Type="Int">-1</Property>
+			<Item Name="Read Default Value.vi" Type="VI" URL="../Read Default Value.vi">
+				<Property Name="NI.ClassItem.ConnectorPane" Type="Bin">&amp;Q#!!!!!!!)!"1!&amp;!!!-!%!!!@````]!!!!"!!%!!!&amp;I!!!!#A!-1#%'=X2B&gt;(6T!!!,1!-!"'.P:'5!!""!-0````]'=W^V=G.F!!!71&amp;!!!Q!!!!%!!AFF=H*P=C"P&gt;81!"!!!!".!#A!.:'6G986M&gt;#"W97RV:1"31(!!(A!!."N49W&amp;O)%6O:WFO:3"49X*J=(2J&lt;G=O&lt;(:M;7)717ZB&lt;'^H)%.I97ZO:7QO&lt;(:D&lt;'&amp;T=Q!!%G&amp;O97RP:S"D;'&amp;O&lt;G6M)'^V&gt;!!!&amp;E"1!!-!!!!"!!)):8*S&lt;X)A;7Y!!&amp;"!=!!?!!!U'V.D97YA27ZH;7ZF)&amp;.D=GFQ&gt;'FO:SZM&gt;GRJ9B:"&lt;G&amp;M&lt;W=A1WBB&lt;GZF&lt;#ZM&gt;G.M98.T!!!297ZB&lt;'^H)'.I97ZO:7QA;7Y!6!$Q!!Q!!Q!%!!5!"A!%!!1!"!!%!!=!"!!%!!A#!!"Y!!!.#!!!!!!!!!E!!!#.#Q!!!!!!!!!!!!!!!!!!!!!!!!A!!!!!!!!!!!!!!*!!!!!!!1!*!!!!!!</Property>
+				<Property Name="NI.ClassItem.ExecutionSystem" Type="Int">-1</Property>
+				<Property Name="NI.ClassItem.Flags" Type="Int">0</Property>
+				<Property Name="NI.ClassItem.IsStaticMethod" Type="Bool">false</Property>
+				<Property Name="NI.ClassItem.MethodScope" Type="UInt">1</Property>
+				<Property Name="NI.ClassItem.Priority" Type="Int">1</Property>
+				<Property Name="NI.ClassItem.State" Type="Int">1342710272</Property>
+			</Item>
+			<Item Name="Write Default Value.vi" Type="VI" URL="../Write Default Value.vi">
+				<Property Name="NI.ClassItem.ConnectorPane" Type="Bin">&amp;Q#!!!!!!!)!"1!&amp;!!!-!%!!!@````]!!!!"!!%!!!&amp;I!!!!#A!-1#%'=X2B&gt;(6T!!!,1!-!"'.P:'5!!""!-0````]'=W^V=G.F!!!71&amp;!!!Q!!!!%!!AFF=H*P=C"P&gt;81!"!!!!&amp;*!=!!?!!!U'V.D97YA27ZH;7ZF)&amp;.D=GFQ&gt;'FO:SZM&gt;GRJ9B:"&lt;G&amp;M&lt;W=A1WBB&lt;GZF&lt;#ZM&gt;G.M98.T!!!397ZB&lt;'^H)'.I97ZO:7QA&lt;X6U!!!71&amp;!!!Q!!!!%!!ABF=H*P=C"J&lt;A!!%U!+!!VE:7:B&gt;7RU)(:B&lt;(6F!&amp;"!=!!?!!!U'V.D97YA27ZH;7ZF)&amp;.D=GFQ&gt;'FO:SZM&gt;GRJ9B:"&lt;G&amp;M&lt;W=A1WBB&lt;GZF&lt;#ZM&gt;G.M98.T!!!297ZB&lt;'^H)'.I97ZO:7QA;7Y!6!$Q!!Q!!Q!%!!1!"1!%!!1!"!!%!!9!"!!(!!A#!!"Y!!!.#!!!!!!!!!!!!!#.#Q!!!!!!!!!!!!!!!!!!!!!!!!A!!!!!!!!!%!!!!*)!!!!!!1!*!!!!!!</Property>
+				<Property Name="NI.ClassItem.ExecutionSystem" Type="Int">-1</Property>
+				<Property Name="NI.ClassItem.Flags" Type="Int">0</Property>
+				<Property Name="NI.ClassItem.IsStaticMethod" Type="Bool">false</Property>
+				<Property Name="NI.ClassItem.MethodScope" Type="UInt">1</Property>
+				<Property Name="NI.ClassItem.Priority" Type="Int">1</Property>
+				<Property Name="NI.ClassItem.State" Type="Int">1342710272</Property>
+			</Item>
+		</Item>
+	</Item>
+	<Item Name="Read Analog Channel.vi" Type="VI" URL="../Read Analog Channel.vi">
```

Comment:
This is the only VI the `Analog Channel.lvclass` claims to own, so everything using Analog Channel is broken on my machine.